app/equipment/armour.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import annotations
+
+
+class Armour:
+    def __init__(self, part: str, protection: int) -> None:
+        self.part = part
+        self.protection = protection
+
+    @staticmethod
+    def set_of_armour(armour_list: list[dict]) -> list[Armour]:
+        return [
+            Armour(armour_part["part"], armour_part["protection"])
```

Comment:
Ensure that each dictionary in `armour_list` contains the keys `part` and `protection` to avoid potential `KeyError`. Consider adding validation or error handling for missing keys.